src/extensionConflicts.ts

```diff
@@ -7,6 +7,7 @@ import { commands, Extension, extensions, window } from 'vscode';
 // A set of VSCode extension ID's that conflict with our extension
 const conflictingIDs = [
   'haaaad.ansible',
+  'lextudio.restructuredtext', // https://github.com/vscode-restructuredtext/vscode-restructuredtext/issues/286
```

Comment:
Side note: do we have a rule for the inline code comment spacing? I usually follow the one from Python — at least two spaces before the comment start.

Reply:
I am not aware of one but you are welcome to dig into eslint options and if there is one check I would agree to enable it. Without it, i would find it not to worth our time.